hasura/metadata/databases/default/tables/public_app.yaml

```diff
@@ -19,6 +19,11 @@ computed_fields:
       function:
         name: get_app_is_verified
         schema: public
+  - name: show_details_page
+    definition:
```

Comment:
what is this doing?

Reply:
A field that is used in the app to decide to open the details page or just go directly to the link.